packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -67,10 +67,11 @@
 }
 
 // Empty / default block side inserter.
-.block-editor-block-list__empty-block-inserter, // Empty paragraph
-.block-editor-default-block-appender .block-editor-inserter { // Empty appender
+.components-popover__content > .block-editor-block-list__empty-block-inserter, // Empty paragraph, needs specificity.
```

Comment:
Do you happen to know where this popover is rendered? or maybe can we find a different way to increase the specificity without using classNames not rendered by the component itself?

Reply:
You could probably double the specificity. I.e. .block-editor-block-list__empty-block-inserter.block-editor-block-list__empty-block-inserter, that might work.